app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  after_save :update_comments_counter
+
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] OPTIONAL
Following best practices `update_comments_counter` should be marked as private after implementing the `after_save` method.